Added info about creating a empty section (#5966)
umbraco/UmbracoDocs: 14/umbraco-cms/extending-backoffice/extension-types/sections-and-trees/README.md

<figure><img src="../../../.gitbook/assets/section.svg" alt=""><figcaption><p>Section</p></figcaption></figure>

**Manifest**

> When creating a new section it's a good idea to use a [Entry Point](../extension-types/entry-point)-extension in your [Umbraco Package Manifest](../../package-manifest) to get better control over all the additional extensions required for the new section.

Reviewer: ```suggestion
When creating a new section it's recommended to use a [Entry Point](../extension-types/entry-point)-extension in your [Umbraco Package Manifest](../../package-manifest). This is to get better control over all the additional extensions required for the new section.
```

This is how to define a section in TypeScript:

```typescript
// TODO: get interface
const section : ManifestSection = {
	type: "section",
	alias: "My.Section",
	name: "My Section",
	meta: {
		"label": "My Section",
		"pathname": "my-section"
	}
}
```

**Manifest with empty element**
If you prefer a clean section you can use an element with a default export, and load the element like this:

```typescript
const section : ManifestSection = {
    type: "section",
    alias: "Empty.Section",
    name : 'Empty Section',
    element : () => import('./empty-section.element.js'),
    meta : {
        label : 'Empty Section',
        pathname : 'empty-section'
    }
}
```

> Notice that if you pass an element like this, the default behavior of loading dashboards and sectionViews will be disabled.

Reviewer: ```suggestion
{% hint style="warning" %}
If you pass an element like this, the default behavior of loading dashboards and sectionViews will be disabled.{% endhint %}
```

}
```

**Manifest with empty element**
Create a element with a default export and load the element like this:

```typescript
const section : ManifestSection = {
    "type": "section",
    "alias": "Empty.Section",
    "name" : 'Empty Section',
    element : () => import('./empty-section.element.js'),
    meta : {
        label : 'Foo',
        pathname : 'empty-section'
    }

> }

Reviewer: I keep trying to add this example in umbraco-package.json however I cannot find a way to make this code work 🤔 

Can you please provide a full example from your package manifest to see how one can reproduce this on a new installation?

Author: Ohh, sorry, this was added using a ”entity point” so it’s actually typescript code. Sorry for the lack of context.

I’ll see if I can update it to make more sense.
